charts/vh-user-api/values.yaml

```diff
@@ -26,6 +26,30 @@ java:
       secrets:
         - azuread--clientsecret
         - azuread--clientid
+        - DefaultPassword
+        - Testing--ExistingGroups--0--Displayname
+        - Testing--ExistingGroups--0--GroupId
+        - Testing--ExistingGroups--1--Displayname
+        - Testing--ExistingGroups--1--Testing--ExistingGroups--0--GroupId
+        - Testing--ExistingUserDisplayname
+        - Testing--ExistingUserFirstname
+        - Testing--ExistingUserId
+        - Testing--ExistingUserLastname
+        - Testing--ExistingUserPrincipal
+        - Testing--ExistingEmail
+        - Testing--NewGroups--0--Displayname
+        - Testing--NewGroups--0--GroupId
```

Comment:
I would remove these test ones as I don't think they are needed for the app

Reply:
Removed.